<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTableView" name="tableView_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>57</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Období zániku</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Okres</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Způsob zániku</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Stav</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="cb_obdobi">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>- - - - - - - - - -</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="cb_okresy">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>- - - - - - - - - -</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="cb_zpusob">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>- - - - - - - - - -</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QComboBox" name="cb_stav">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>- - - - - - - - - -</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Soubor">
    <property name="title">
     <string>&amp;Soubor</string>
    </property>
    <addaction name="action_Pripoj_databazi"/>
    <addaction name="action_Export_databaze"/>
    <addaction name="action_Konec"/>
   </widget>
   <widget class="QMenu" name="menu_Vlo_it">
    <property name="title">
     <string>&amp;Vložit</string>
    </property>
    <addaction name="action_Nov"/>
   </widget>
   <addaction name="menu_Soubor"/>
   <addaction name="menu_Vlo_it"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Konec">
   <property name="text">
    <string>&amp;Konec</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Nov">
   <property name="text">
    <string>&amp;Nový</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Pripoj_databazi">
   <property name="text">
    <string>&amp;Pripoj databazi</string>
   </property>
  </action>
  <action name="action_Export_databaze">
   <property name="text">
    <string>&amp;Export databaze</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
